PyU4V/tools/openstack/migrate_utils.py

```diff
@@ -0,0 +1,866 @@
+# Copyright (c) 2019 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OpenStack migrate_utils.py.
+
+Migrate utilities for OpenStack
+"""
+
+from __future__ import print_function
+
+from builtins import input
+import logging
+import re
+import sys
+
+from PyU4V.utils import exception
+
+import prettytable
+
+LOG = logging.getLogger(__name__)
+
+YES_CONSTANTS = ['y', 'yes']
+NO_CONSTANTS = ['n', 'no']
+EXIT_CONSTANTS = ['x', 'exit']
+
+VERSION = '2.0.0'
+
+DEBUG = 'DEBUG'
+INFO = 'INFO'
+WARNING = 'WARNING'
+ERROR = 'ERROR'
+
+
+class MigrateUtils(object):
+    """OpenStack migrate utils."""
+
+    def __init__(self, conn):
+        """Initialize class.
+
+        :param conn: the request connection
+        """
+        self.conn = conn
+
+    def check_input(self, txt_str, option):
```

Comment:
Method is static, will we keep conformity with the rest of PyU4V and mark static methods as static? There are a number of them in this file but ill keep it to this one comment about the collection of them as a whole

Reply:
@staticmethod decorator needed